go.mod

```diff
@@ -65,7 +65,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2
-	github.com/tikv/client-go/v2 v2.0.0-alpha.0.20211221095044-38d30d5632a8
+	github.com/tikv/client-go/v2 v2.0.0-alpha.0.20220601055058-46205016d898
```

Comment:
There's a new pick in the client-go v5.3 branch. https://github.com/tikv/client-go/commits/tidb-5.3